model_utils/tracker.py

```diff
@@ -63,7 +63,7 @@ def changed(self):
 
     def init_deferred_fields(self):
         self.instance._deferred_fields = []
-        if not self.instance._deferred:
+        if hasattr(self.instance, '_deferred') and not self.instance._deferred:
```

Comment:
eh, doesn't matter, that's shorter but might be less readable. I'll merge it as-is